src/search/index.ts

```diff
@@ -1,8 +1,30 @@
+import * as digit from "./digit";
+import * as spec from "./spec";
+
+const searchers: spec.Interface[] = [digit.Searcher];
```

Comment:
This is the list of searchers we just extend. The abstract searcher implementation below should usually not be touched anymore.